src/components/lastUpdated/LastUpdated.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as moment from 'moment';
+
+export interface ILastUpdatedOwnProps extends React.ClassAttributes<LastUpdated> {
+  id: string;
+}
+
+export interface ILastUpdatedProps extends ILastUpdatedOwnProps {
+  label?: string;
+  time?: Date;
+  onRender?: () => void;
+  onDestroy?: () => void;
+}
+
+export const defaultLastUpdateLabel: string = 'Last update:';
+
+export class LastUpdated extends React.Component<ILastUpdatedProps, any> {
+
+  componentDidMount() {
+    if (this.props.onRender) {
+      this.props.onRender();
```

Comment:
My personal preference here:

I think the app should be "data driven". What I mean is that component that is using the LastUpdatedConnected component should set the state in the store and the LastUpdatedConnected should read this state and render. See https://github.com/coveo/react-vapor/blob/master/docs/members-example/MembersExample.tsx. 

Being "data driven" means it's not the LastUpdated components job to mount itself in the store and that the LastUpdated component won't need a onRender prop.

@GermainBergeron  you might want to argue on this one since you stated the "data driven" way of using Redux in the Admin.